ONPRC_EHR_ComplianceDB/resources/schemas/dbscripts/sqlserver/onprc_ehr_compliancedb-0.00-24.00.sql

```diff
@@ -0,0 +1,41 @@
+
+CREATE SCHEMA onprc_ehr_compliancedb;
+GO
+
+CREATE TABLE onprc_ehr_compliancedb.SciShield_Data
+(
+    RowId INT IDENTITY(1,1) NOT NULL,
+    employeeId nvarchar(255) not null,
+    requirementname nvarchar(255) null,
+    Date datetime null,
+    Container ENTITYID NOT NULL,
+    comment nvarchar(2000) null,
+    CreatedBy USERID,
+    Created datetime,
+    ModifiedBy USERID,
+    Modified datetime,
+    processed int NULL
+
+    CONSTRAINT PK_ScieShield_Data PRIMARY KEY (RowId),
+    CONSTRAINT FK_ONPRC_EHR_COMPLIANCE_SCISHIELD_DATA_CONTAINER FOREIGN KEY (Container) REFERENCES core.Containers (EntityId)
```

Comment:
Since both tables have a Container column with a FK, they should also have an index.